grails-app/services/au/org/ala/bie/SearchService.groovy

```diff
@@ -141,7 +141,7 @@ class SearchService {
             q = "*:*"
             queryTitle = "all records"
         }
-        def response = indexService.search(true, q, fqs, requestedFacets, start, rows, params.sort, params.dir, params.sort2, params.dir2, params?.q_op)
```

Comment:
Instead of getting flimit from params (which comes from the user), pass flimit in to the method as an optional argument.